core/remote/change.js

```diff
@@ -460,3 +461,22 @@ function sort(changes /*: Array<RemoteChange> */) /*: Array<RemoteChange> */ {
   // return changes.sort(sortByPath).sort(sortByAction)
   return changes.sort(sortChanges)
 }
+
+function sortByPath(
+  changes /*: Array<RemoteChange> */
+) /*: Array<RemoteChange> */ {
+  return changes.sort((
+    { doc: aDoc } /*: RemoteChange */,
+    { doc: bDoc } /*: RemoteChange */
+  ) => {
+    if (!aDoc.path) return 1
+    if (!bDoc.path) return -1
+
+    const aPath = aDoc.path.normalize()
+    const bPath = bDoc.path.normalize()
+
+    if (aPath < bPath) return -1
+    if (bPath > aPath) return 1
```

Comment:
`aPath < bPath` and `bPath > aPath` are two ways to write the same condition, which means that the `return 1` here is dead code

Reply:
Good catch. I'll fix this right away.